core/src/main/java/com/pholser/junit/quickcheck/internal/Reflection.java

```diff
@@ -265,6 +259,13 @@ private static RuntimeException reflectionException(Exception ex) {
         return new ReflectionException(ex);
     }
 
```

Comment:
Could you change this method's name to `nonSystemAnnotations`? Reads more consistently with `allAnnotations`, `annotatedComponentTypes`, etc. I don't favor the JavaBeans conventions where not strictly necessary.